README.md

```diff
@@ -14,7 +16,7 @@ The contents of this code of conduct apply to our interactions in various areas
 
 We want our team to be fun, productive, and safe for all members. We've adopted the Social Tables [core values](https://www.socialtables.com/company/about/#values), in ways that specifically apply to the craft of Software Engineering at Social Tables:
 
-**We recognize that "It Takes a Village" as we work alongside other teams.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—indeed, friendships—with our colleagues. That collaboration is core to our success.
+**We recognize that "It Takes a Village" as we work alongside other teams.** Customer Success, Sales, Marketing, and Operations team members and others across the company are our peers and collaborators, not our clients. We maintain open lines of communication and foster good working relationships—-indeed, friendships—-with our colleagues. That collaboration is core to our success.
```

Comment:
@cpolitano Did you mean to change this to an m-dash followed by a hyphen? I think we need to just do standardization pass on things like curly quotes, m-dash vs. two hyphens, etc.